nilearn/datasets/struct.py

```diff
@@ -118,6 +120,75 @@ def load_mni152_template():
     return check_niimg(path)
 
 
+def load_mni152_brain_mask():
+    """Load brain mask from MNI152 template
+
+    Returns
+    -------
+    mask_img: Nifti-like mask image corresponding to grey and white matter.
+
+    References
+    ----------
+    Refer to load_mni152_template function for more information about MNI
+    template
+    """
+    # Load MNI template
+    target_img = load_mni152_template()
+    mask_voxels = (target_img.get_data() > 0).astype(int)
+    mask_img = new_img_like(target_img, mask_voxels)
+    return mask_img
+
+
+def fetch_mni152_grey_matter_mask(data_dir=None, resume=True, verbose=1):
+    """Downloads ICBM152 template first, then loads computed 'gm' mask image.
+
+    The output 'gm' mask image will be resampled to MNI152 template space.
```

Comment:
> What if somebody want to use in their own experiments ?

They resample. The nifti masker does that automatically. No need to
resample twice.